src/pages/offer-details/OfferDetails.tsx

```diff
@@ -1,10 +1,67 @@
+import { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useParams, useNavigate } from 'react-router-dom'
+
 import Box from '@mui/material/Box'
 import Container from '@mui/material/Container'
 
+import OfferCard from '~/components/offer-card/OfferCard'
+import { defaultResponses } from '~/constants'
+import useAxios from '~/hooks/use-axios'
+import { mockOffer } from '~/pages/find-offers/FindOffers.constants'
+import { OfferService } from '~/services/offer-service'
+import AppCard from '~/components/app-card/AppCard'
+import { errorRoutes } from '~/router/constants/errorRoutes'
+import { styles } from '~/pages/offer-details/OfferDetails.styles'
+import Loader from '~/components/loader/Loader'
+
 const OfferDetails = () => {
+  const { t } = useTranslation()
+  const { id } = useParams()
+  const navigate = useNavigate()
+
+  const getOffer = useCallback(() => OfferService.getOffer(id), [id])
+  const { response, loading } = useAxios({
```

Comment:
you don`t use this response

Reply:
I know about it. We will use it in the next tasks. But I deleted it.